packages/start-plugin-core/tests/importProtection/analysis.exports.test.ts

```
@@ -1,10 +1,10 @@
 import { describe, expect, test } from 'vitest'
 
 import {
-  collectNamedExports,
-  collectMockExportNamesBySource,
   isValidExportName,
-} from '../../src/import-protection-plugin/rewriteDeniedImports'
+  getMockExportNamesBySource,
```

ESLint

[error] Member 'getMockExportNamesBySource' of the import declaration should be sorted alphabetically.

(sort-imports)